lib/src/stripe_api.dart

```diff
@@ -91,7 +91,8 @@ class StripeApi {
 
   /// Confirm a PaymentIntent
   /// https://stripe.com/docs/api/payment_intents/confirm
-  Future<Map<String, dynamic>> confirmPaymentIntent(String clientSecret, {Map<String, dynamic> data = const {}}) async {
```

Comment:
Would it be enough to just remove the `const` modifier?

Reply:
That was what I originally wanted to write, but was not possible.
`error: The default value of an optional parameter must be constant. (non_constant_default_value at [stripe_sdk] lib\src\stripe_api.dart:113)`